md/groovy-in-bonita.md

```diff
@@ -0,0 +1,347 @@
+# Groovy script language in Bonita
+
+In Bonita, it is a prerequisite to be comfortable with Java when implementing advanced behaviors in your processes, the below article aims at helping with Groovy, the scripting language used by Bonita.
+Groovy has been chosen as the scripting language because it is :
+* Concise, readable and have an expressive syntax, easy to learn for Java developers
+* Seamlessly and transparently integrates and interoperates with Java and any third-party libraries
```

Comment:
Almost true, you can't use native java lambda syntax, you must transform it to a closure.